homeassistant/components/renault/config_flow.py

```diff
@@ -0,0 +1,108 @@
+"""Config flow to configure Renault component."""
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME
+
+from .const import (  # pylint: disable=unused-import
+    AVAILABLE_LOCALES,
+    CONF_GIGYA_APIKEY,
+    CONF_KAMEREON_ACCOUNT_ID,
+    CONF_KAMEREON_APIKEY,
+    CONF_LOCALE,
+    DOMAIN,
+)
+from .pyzeproxy import PyzeProxy
+
+DEFAULT_GIGYA_APIKEY = "3_e8d4g..."
+DEFAULT_KAMEREON_APIKEY = "oF09WnK..."
```

Comment:
Why having defaults if this is not working with this values.

Default should be usable, don't use it as placeholders.